clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -3454,10 +3464,11 @@ void Sema::CheckSYCLKernelCall(FunctionDecl *KernelFunc, SourceRange CallLoc,
   if (!LangOpts.SYCLIsDevice)
     return;
 
-  const CXXRecordDecl *KernelObj =
+  CXXRecordDecl *KernelObj =
       GetSYCLKernelObjectType(KernelFunc)->getAsCXXRecordDecl();
 
-  if (!KernelObj) {
+  CXXMethodDecl *CallOperator = IsCallOperatorDefined(KernelObj);
+  if (!CallOperator) {
```

Comment:
```suggestion
  if (!IsCallOperatorDefined(KernelObj)) {
```
(and change the name to whatever we call it above).